RELEASE_NOTES.md

```diff
@@ -1,26 +1,36 @@
 # Monitoring as Code Tool - Release Notes
 
 - [Monitoring as Code Tool - Release Notes](#monitoring-as-code-tool---release-notes)
-  - [1.3.0](#130)
+  - [1.3.1](#131)
     - [List of changes](#list-of-changes)
+      - [Bugfixes](#bugfixes)
+  - [1.3.0](#130)
+    - [List of changes](#list-of-changes-1)
       - [Deprecation Warnings](#deprecation-warnings)
       - [Features](#features)
-      - [Bugfixes](#bugfixes)
+      - [Bugfixes](#bugfixes-1)
```

Comment:
Maybe we should introduce some `<a id=""/>` tags to link to them instead of
updating the whole index all the time.

Reply:
Good idea! Or we remove the subheadings from the toc.

Reply:
Would also work :laughing: